packages/react/test/specs/components/Chat/ChatMessage-test.tsx

```diff
@@ -18,9 +17,7 @@ describe('ChatMessage', () => {
   chatMessageImplementsShorthandProp('content', Box, { mapsValueToProp: 'children' })
 
   describe('accessibility', () => {
-    handlesAccessibility(ChatMessage, {
-      focusZoneDefinition: (chatMessageBehavior as AccessibilityDefinition).focusZone,
```

Comment:
This thing was broken as `chatMessageBehavior` is a function, there is no `focusZone` prop